.github/workflows/reusable-workflow-e2e.yaml

```diff
@@ -313,7 +313,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
         with:
-          repository: openstack-k8s-operators/openstack-k8s-operators-ci
+          repository: tripleo-ci/openstack-k8s-operators-ci
```

Comment:
This is for testing only, I will update this PR and remove thse changes after testing in tripleo-ci org.